test-benchmark/setup-frame.html

```diff
@@ -0,0 +1,111 @@
+<!--
+@license
+Copyright (c) 2015 The Polymer Project Authors. All rights reserved.
+This code may only be used under the BSD style license found at http://polymer.github.io/LICENSE.txt
+The complete set of authors may be found at http://polymer.github.io/AUTHORS.txt
+The complete set of contributors may be found at http://polymer.github.io/CONTRIBUTORS.txt
+Code distributed by Google as part of the polymer project is also
+subject to an additional IP rights grant found at http://polymer.github.io/PATENTS.txt
+-->
+
+<script>
+(function() {
+  'use strict';
+
+  /**
+   * The SetupFrame function represents the harness that is provided in every
+   * benchmark test. The harness is created by stringifying the SetupFrame
+   * function and converting it to an IIFE within the `<head>` of the final
+   * test run document.
+   */
+  function SetupFrame() {
+    'use strict';
+
+    /**
+     * Test
```

Comment:
Can't you just start with "A test instance..." below, like we do for every other comment? :)